jetty-core/jetty-deploy/src/test/resources/etc/webdefault.xml

```diff
@@ -52,15 +52,15 @@
   <!--
     UNCOMMENT TO ACTIVATE 
     <context-param> 
-      <param-name>org.eclipse.jetty.ee9.servlet.SessionDomain</param-name>
+      <param-name>org.eclipse.jetty.servlet.SessionDomain</param-name>
       <param-value>127.0.0.1</param-value> 
     </context-param> 
     <context-param>
-      <param-name>org.eclipse.jetty.ee9.servlet.SessionPath</param-name>
+      <param-name>org.eclipse.jetty.servlet.SessionPath</param-name>
       <param-value>/</param-value>
     </context-param>
     <context-param>
-      <param-name>org.eclipse.jetty.ee9.servlet.MaxAge</param-name>
+      <param-name>org.eclipse.jetty.servlet.MaxAge</param-name>
```

Comment:
Are these parameters really missing the `eeN` in the package name?

Reply:
No, all session code except for `SessionHandler` is environment agnostic, and thus the config params.